src/Table/Table.php

```diff
@@ -499,6 +499,7 @@ public function render()
     /**
      * Magic method which returns the rendered table
      *
+     * @psalm-suppress MethodSignatureMustProvideReturnType
```

Comment:
This should be baselined, not suppressed

Reply:
Made the suggested changes and pushed the code. Kindly review.